receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper.go

```diff
@@ -63,6 +64,13 @@ func (s *scraper) scrape(_ context.Context) (pdata.MetricSlice, error) {
 		return metrics, scrapererror.NewPartialScrapeError(err, metricsLen)
 	}
 
+	if s.config.PerCPU {
+		divisor := float64(runtime.NumCPU())
+		avgLoadValues.Load1 = avgLoadValues.Load1 * divisor
```

Comment:
Shouldn't you be dividing, instead of multiplying?

Reply:
obviously I was tired ^^ fixed